lib/matplotlib/tests/test_colors.py

```diff
@@ -195,10 +195,6 @@ def test_colormap_equals():
     # Make sure we can compare different sizes without failure
     cm_copy._lut = cm_copy._lut[:10, :]
     assert cm_copy != cmap
-    # Test different names are not equal
-    cm_copy = cmap.copy()
-    cm_copy.name = "Test"
-    assert cm_copy != cmap
```

Comment:
I suggest to keep this and test that colormaps with different names are still equal.